packages/core/src/interpreter.ts

```diff
@@ -913,6 +923,9 @@ export class Interpreter<
     }
   }
   public spawn(entity: Spawnable, name: string, options?: SpawnOptions): Actor {
+    if (this._status !== InterpreterStatus.Running) {
```

Comment:
this change doesn't seem to be described in the changeset nor in added tests, shouldn't we add both for this? A test would let me understand this change better - because right now I'm not sure what exact issue gets fixed by this. What does it give for us to spawn a null actor here? Wouldnt it be maybe more appropriate to throw an error instead?

Reply:
This can be removed for now.